packages/block-library/src/navigation/edit/index.js

```diff
@@ -109,12 +109,18 @@ function Navigation( {
 		layout: { justifyContent, orientation = 'horizontal' } = {},
 	} = attributes;
 
-	const [ areaMenu, setAreaMenu ] = useEntityProp(
-		'root',
-		'navigationArea',
-		'navigation',
-		navigationArea
-	);
+	let areaMenu, setAreaMenu;
```

Comment:
I think we should set a default value for `setAreaMenu`, probably a loadash `noop`, otherwise calling it will trigger an error.